### ISSUE TITLE
[Feature request] Add option to use jspm as bundler

### ISSUE THREAD

**Issue by iscekic** [cafreeman/generator-mobx-react#1]
Would be nice to have.

http://jspm.io/

**Comment by mweststrate** [comment]
See also: https://github.com/capaj/jspm-react

**Comment by iscekic** [comment]
Thanks, I'll check it out.

**Comment by cafreeman** [comment]
@Torwori I've considered giving JSPM a shot but there are currently 2 things I'm waiting on:
- It seems that JSPM 0.17 is really the version I'd be interested in, as it brings a lot of changes, especially with regard to React, but it's not currently stable, so I'm waiting to see how that looks
- Not sure how Webpack2 will affect this. I'll be interested to see how the the two compare

For now, I'm going to keep an eye on both and hold tight until something stabilizes there. In the mean time, it looks like @mweststrate's recommendation will hopefully tide you over :)

**Comment by iscekic** [comment]
@cafreeman You're right, the new versions will be interesting to compare. :) I'll close the issue for now.
